src/transforms/addEslintDisableComment.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Type definitions for VSCode autocompletion!
+ *
+ * @typedef {Object} TransformParams
+ * @property {*} ast - The resulting AST as parsed by babel
+ * @property {import("ast-types/gen/builders").builders} builder - Recast builder for transforming the AST
+ * @property {*} options - Options passed into the transform from the CLI (if any)
+ */
+
+/**
+ * @param {TransformParams} param0
+ * @returns {import("ast-types").Visitor}
+ */
+
+const reactHookPrefix = new RegExp("use\\w+");
+const transform = ({ builder }) => {
+  return {
+    visitExportNamedDeclaration(path) {
+      const includesHook = path.node.specifiers.some(specifier => {
+        if (reactHookPrefix.test(specifier.local.name)) return true;
+      });
```

Comment:
I'm not 100% sure how this is happening but if a query has multiple export statements like so:
```tsx
export type { QueryData };
export { useQuery, Some_Query };
```

Then it adds the eslint disable comment to both of those statements and also converts `export type { QueryData };` to `export { QueryData };` which TypeScript starts to complain about.

Reply:
Wow, I found out this was because the phrase `use` was included in one of the types. 
Great catch Ryan, I'll also put a fix PR the regex in ESLint rules as well.